libs/application_runner/test/environment/query_server_test.exs

```diff
@@ -181,6 +181,7 @@ defmodule ApplicationRunner.Environment.QueryServerTest do
                  "test",
                  Parser.parse!("{}"),
                  Parser.replace_params(%{}, %{}),
+                 %{},
```

Comment:
Can't you make the projections and options optional instead of changing all the unit tests ?